docker-compose.yml

```diff
@@ -1,14 +1,16 @@
-version: "2"
+version: 3
+
 services:
-  koa_ex.mongo:
+  db:
     image: tutum/mongodb:3.0
     ports:
-    - "27017:27017"
-    - "28017:28017"
+      - '27017:27017'
+      - '28017:28017'
     restart: always
     volumes:
-      - koa_ex_db:/data/db
+      - db:/data/db
```

Comment:
If I had many `volumes` or `containers`, is it difficult to distinguish them?

Reply:
No, it prefixes containers with `foldername_` unless you specify `containerName` on each definition

Reply:
For example, in a directory called `myapp`, all the containers will be like `myapp_db_1`, etc.